Sources/TuistKit/ProjectEditor/ProjectEditorMapper.swift

```diff
@@ -248,7 +266,7 @@ final class ProjectEditorMapper: ProjectEditorMapping {
         let scheme = Scheme(name: projectName, shared: true, buildAction: buildAction, runAction: runAction)
         let projectSettings = Settings(
             base: [
-                "ONLY_ACTIVE_ARCH": "NO",
+                "ONLY_ACTIVE_ARCH": "YES",
                 "EXCLUDED_ARCHS": .string(excludedArchs()),
             ],
```

Comment:
Since `YES` is the default (for debug builds) there's no need to explicitly set it. With that done, setting `EXCLUDED_ARCHS` shouldn't be necessary either.

```suggestion
```